parse/IntComplexValueRefParser.cpp

```diff
@@ -102,6 +102,13 @@ namespace parse {
                     ) [ _val = new_<ValueRef::ComplexVariable<int> >(_a, _b, _c, _f, _d, _e) ]
                 ;
 
+            ship_parts_owned
```

Comment:
not necessary now, but planning to add a parameter to specify a part type would be good